specification/iotoperations/IoTOperations.Management/client.tsp

```diff
@@ -5,3 +5,12 @@ using Azure.ClientGenerator.Core;
 
 @@clientName(Microsoft.IoTOperations, "IoTOperationsMgmt", "python");
 @@clientName(Microsoft.IoTOperations, "IoTOperationsManagementClient", "java");
+
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+namespace Microsoft.IoTOperations {
+  @@access(Operations.list, Access.internal);
```

Comment:
This workaround is not needed anymore, generator can remove the list operation properly.